automation/usersAPI/deploy-users-templet.ps1

```diff
@@ -0,0 +1,68 @@
+[CmdletBinding()]
+param (
+    [Parameter(Mandatory=$True)]
+    [string]
+    $subscriptionId,
+
+    [Parameter(Mandatory=$True)]
+    [string]
+    $tenantId,
+    
+
+    [Parameter(Mandatory=$True)]
+    [string]
+    $servicePrincipalId,
+
+   
+    [Parameter(Mandatory=$true)]
+    [ValidateNotNullOrEmpty()]
+    [string]$servicePrincipalPassword=$(Throw "Password required."),
+   
+    
+    [Parameter(Mandatory=$True)]
+    [string]
+    $resourceGroupName,
+
+    [Parameter(Mandatory=$True)]
+    [string]
+    $functionName,
+
+    [Parameter(Mandatory=$True)]
+    [string]
+    $storageAccountName,
+
+    [Parameter(Mandatory=$True)]
+    [string]
+    $appServicePlanName,
+
+    [Parameter(Mandatory=$True)]
+    [string]
+    $location,
+
+    [Parameter(Mandatory=$True)]
+    $templateFilePath
+)
+
+#open log
+$prefix = "yergu-Arm-deplo"
+$stamp = (Get-Date).toString().Replace("/","-").Replace(":","-")
+Start-Transcript "./$prefix-$stamp.log"
```

Comment:
I like what you did here to get a timestamp for the log file name but is there a better way to format this than doing a match and replace?

Reply:
I am not sure I will figure out